cmd/operator/options/options.go

```diff
@@ -80,6 +81,8 @@ func New() *Options {
 	flag.IntVar(&opts.APIPort, "port", 6500, "The port for the operator API server to listen on")
 	flag.IntVar(&opts.HealthzPort, "healthz-port", 8080, "The port for the healthz server to listen on")
 
+	flag.Var(&opts.AdditionalCPServices, "additional-control-plane-services", "Name of the additional control plane services, if any")
```

Comment:
Nit: should this be `additional-control-plane-service` since each value can contain a single service?